components/mailer.rst

```diff
@@ -142,6 +142,17 @@ The mailer will start using the first transport and if it fails, it will retry
 the same delivery with the next transports until one of them succeeds (or until
 all of them fail).
 
+Disable TLS peer verification
```

Comment:
TLS peer verification

Reply:
As this is a title, we prefer:
```suggestion
TLS Peer Verification
```

Please adjust the underline length accordingly 👍